network/endpoint.go

```diff
@@ -109,8 +109,8 @@ type EndpointInfo struct {
 	Options                       map[string]interface{}
 	DisableHairpinOnHostInterface bool
 	IsIPv6Enabled                 bool
-
-	HostSubnetPrefix string // can be used later to add an external interface
+	PnPID                         string
```

Comment:
can we add a brief comment about the pnp here or in interface info-- the pnpid isn't needed later during deletion right? and does backend nic only exist in windows stateless cni?